[2022-08-09 03:20:37,607] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-08T00:00:00+00:00 [queued]>
[2022-08-09 03:20:37,619] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-08T00:00:00+00:00 [queued]>
[2022-08-09 03:20:37,620] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-09 03:20:37,620] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-09 03:20:37,620] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-09 03:20:37,638] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): final_dag> on 2022-08-08 00:00:00+00:00
[2022-08-09 03:20:37,645] {standard_task_runner.py:52} INFO - Started process 10362 to run task
[2022-08-09 03:20:37,648] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'final_dag', 'scheduled__2022-08-08T00:00:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmp3e36wrhh', '--error-file', '/tmp/tmppo1utzo4']
[2022-08-09 03:20:37,649] {standard_task_runner.py:80} INFO - Job 152: Subtask final_dag
[2022-08-09 03:20:37,721] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.final_dag scheduled__2022-08-08T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-09 03:20:37,884] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=final_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-08T00:00:00+00:00
[2022-08-09 03:20:37,989] {python.py:173} INFO - Done. Returned value was: None
[2022-08-09 03:20:38,005] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=final_dag, execution_date=20220808T000000, start_date=20220809T032037, end_date=20220809T032038
[2022-08-09 03:20:38,061] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-09 03:20:38,095] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
